specification/_doc_ids/table.csv

```diff
@@ -0,0 +1,381 @@
+paginate-search-results,/guide/en/elasticsearch/reference/current/paginate-search-results.html
+sort-search-results,/guide/en/elasticsearch/reference/current/sort-search-results.html
+query-dsl-mlt-query,/guide/en/elasticsearch/reference/current/query-dsl-mlt-query.html
+query-dsl,/guide/en/elasticsearch/reference/current/query-dsl.html
+common-options,/guide/en/elasticsearch/reference/current/common-options.html
+indices-create-data-stream,/guide/en/elasticsearch/reference/current/indices-create-data-stream.html
+mapping-date-format,/guide/en/elasticsearch/reference/7.x/mapping-date-format.html
+query-dsl-minimum-should-match,/guide/en/elasticsearch/reference/current/query-dsl-minimum-should-match.html
+query-dsl-multi-term-rewrite,/guide/en/elasticsearch/reference/current/query-dsl-multi-term-rewrite.html
+modules-node,/guide/en/elasticsearch/reference/current/modules-node.html
+search-aggregations-bucket-count-ks-test-aggregation,/guide/en/elasticsearch/reference/current/search-aggregations-bucket-count-ks-test-aggregation.html
+search-aggregations-bucket-correlation-aggregation,/guide/en/elasticsearch/reference/current/search-aggregations-bucket-correlation-aggregation.html
+search-aggregations-bucket-categorize-text-aggregation,/guide/en/elasticsearch/reference/current/search-aggregations-bucket-categorize-text-aggregation.html
+,/guide/en/machine-learning/7.12/ml-feature-importance.html
+analysis-normalizers,/guide/en/elasticsearch/reference/current/analysis-normalizers.html
+geo-shape,/guide/en/elasticsearch/reference/current/geo-shape.html
+shape,/guide/en/elasticsearch/reference/current/shape.html
+mapping-meta-field,/guide/en/elasticsearch/reference/current/mapping-meta-field.html
+query-dsl-simple-query-string-query,/guide/en/elasticsearch/reference/7.15/query-dsl-simple-query-string-query.html
+modules-remote-clusters,/guide/en/elasticsearch/reference/current/modules-remote-clusters.html
+cluster-reroute,/guide/en/elasticsearch/reference/current/cluster-reroute.html
+modules-cluster,/guide/en/elasticsearch/reference/current/modules-cluster.html
+modules-fielddata,/guide/en/elasticsearch/reference/current/modules-fielddata.html
+mapping,/guide/en/elasticsearch/reference/current/mapping.html
+analyzer-anatomy,/guide/en/elasticsearch/reference/current/analyzer-anatomy.html
+modules-discovery-hosts-providers,/guide/en/elasticsearch/reference/current/modules-discovery-hosts-providers.html
+important-settings,/guide/en/elasticsearch/reference/current/important-settings.html
+cluster,/guide/en/elasticsearch/reference/current/cluster.html
+eql-syntax,/guide/en/elasticsearch/reference/current/eql-syntax.html
+eql,/guide/en/elasticsearch/reference/current/eql.html
+index-modules,/guide/en/elasticsearch/reference/7.8/index-modules.html
+,/guide/en/machine-learning/current/dfa-classification.html
+,/guide/en/machine-learning/current/dfa-regression.html
+,/guide/en/machine-learning/7.12/dfa-regression.html
```

Comment:
Since we have a keyword for the "current" version of this page, I think a 7.12-specific one is unnecessary
```suggestion

```